fastify.js

```diff
@@ -259,6 +259,10 @@ function build (options) {
     return fastify
   }
 
+  function exposeRoutes () {
+    return map.entries()
+  }
```

Comment:
[Map](https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Global_Objects/Map) does not have a method that returns an iterable with objects, just the array `[key, value]` or only `keys`, or only `values`.
I do not know if it is worth the effort create an iterator like the one you are proposing.